src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.test.js

```diff
@@ -470,6 +520,114 @@ describe('Checkout component', () => {
 
       const inputPhone = container.querySelector('input#phone');
       fireEvent.change(inputPhone, { target: { value: '+54 223 655-8877' } });
+
+      const selectSecurityQuestion = container.querySelector('select#securityquestion');
+      fireEvent.change(selectSecurityQuestion, { target: { value: '1' } });
+
+      const inputAnswerSecurityQuestion = container.querySelector('input#answerSecurityQuestion');
+      fireEvent.change(inputAnswerSecurityQuestion, { target: { value: 'Answer' } });
+    });
+
+    await act(async () => {
+      const submitButton = container.querySelector('button[type="submit"]');
+      fireEvent.submit(submitButton);
+    });
+
+    // Assert
+    expect(getByText('validation_messages.error_required_field')).toBeInTheDocument();
+  });
+
+  it('should show error message if security question field is empty', async () => {
+    // Arrange
+    let result;
+    await act(async () => {
+      result = render(<ContactInformationElement />);
+    });
+
+    const { container, getByText } = result;
+
+    // Act
+    act(() => {
+      const inputFirstname = container.querySelector('input#firstname');
+      fireEvent.change(inputFirstname, { target: { value: 'Test' } });
+
+      const inputLastname = container.querySelector('input#lastname');
+      fireEvent.change(inputLastname, { target: { value: 'Test' } });
+
+      const inputAddress = container.querySelector('input#address');
+      fireEvent.change(inputAddress, { target: { value: 'address 1' } });
+
+      const inputCity = container.querySelector('input#city');
+      fireEvent.change(inputCity, { target: { value: 'tandil' } });
+
+      const selectCountry = container.querySelector('select#country');
+      fireEvent.change(selectCountry, { target: { value: 'ar' } });
+
+      const selectProvince = container.querySelector('select#province');
+      fireEvent.change(selectProvince, { target: { value: 'AR-B' } });
+
+      const selectIndustry = container.querySelector('select#industry');
+      fireEvent.change(selectIndustry, { target: { value: 'dplr1' } });
+
+      const inputPhone = container.querySelector('input#phone');
+      fireEvent.change(inputPhone, { target: { value: '+54 223 655-8877' } });
+
+      const selectSecurityQuestion = container.querySelector('select#securityquestion');
+      fireEvent.change(selectSecurityQuestion, { target: { value: '' } });
+
+      const inputAnswerSecurityQuestion = container.querySelector('input#answerSecurityQuestion');
+      fireEvent.change(inputAnswerSecurityQuestion, { target: { value: 'Answer' } });
+    });
+
+    await act(async () => {
+      const submitButton = container.querySelector('button[type="submit"]');
+      fireEvent.submit(submitButton);
+    });
+
+    // Assert
+    expect(getByText('validation_messages.error_required_field')).toBeInTheDocument();
+  });
+
+  it('should show error message if answer field is empty', async () => {
+    // Arrange
```

Comment:
This test is duplicated. It's not necessary because already exists.

Reply:
@jcamposmk Reviewing the existing code, I found that the industry selection had its own test and followed that line, but if you think it is redundant, I can remove it